checkov/terraform/checks/resource/aws/ElasticsearchNodeToNodeEncryption.py

```diff
@@ -23,12 +24,12 @@ def scan_resource_conf(self, conf):
                 if "instance_count" not in cluster_config:
                     return CheckResult.PASSED
                 self.evaluated_keys = ['cluster_config/[0]/instance_count']
-                instance_count = cluster_config["instance_count"]
-                if isinstance(instance_count, int):
+                instance_count = force_int(cluster_config["instance_count"][0])
```

Comment:
Check the list is indeed not empty with one element in it

Reply:
can it be empty?

Reply:
@tronxd Please approve changes.